platform/admin/core/admin-core-impl/src/test/java/org/codice/ddf/admin/core/impl/AdminAlertImplTest.java

```diff
@@ -0,0 +1,144 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.admin.core.impl;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.shiro.subject.PrincipalCollection;
+import org.apache.shiro.util.ThreadContext;
+import org.codice.ddf.persistence.PersistenceException;
+import org.codice.ddf.persistence.PersistentStore;
+import org.codice.ddf.system.alerts.Alert;
+import org.codice.ddf.system.alerts.SystemNotice;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.osgi.service.event.Event;
+import org.osgi.service.event.EventAdmin;
+
+import ddf.security.Subject;
+
+@RunWith(MockitoJUnitRunner.class)
+public class AdminAlertImplTest {
+
+    @Mock
+    PersistentStore persistentStore;
+
+    @Mock
+    EventAdmin eventAdmin;
+
+    @Mock
+    Subject subject;
+
+    @Mock
+    PrincipalCollection principals;
+
+    AdminAlertImpl adminAlert;
+
+    Map<String, Object> solrMap;
+
+    @Before
+    public void setup() throws Exception {
+        adminAlert = new AdminAlertImpl(persistentStore, eventAdmin);
+        solrMap = new HashMap<>();
+        solrMap.put("id_txt", "myId");
```

Comment:
✏️  I find it a little weird that below you have `SystemNotice.SYSYTEM_NOTICE_DETAILS_KEY + "_txt"` but here you have it inlined.